src/store/participantsStore.js

```diff
@@ -56,6 +56,20 @@ function emitUserStatusUpdated(participant) {
 	}
 }
 
+/**
+ * Get connection failed error message as per error code.
+ *
+ * @param {*} code Error message code
+ *
+ * @return {string} Error message
+ */
+function getConnectionFailedErrorMessage(code) {
+	if (code === 'consent') {
+		return t('spreed', 'Recording consent is required')
+	}
+	return t('spreed', 'Oops, something went wrong!')
```

Comment:
I'd keep it more formal, and easier to translate to other languages
```suggestion
	return t('spreed', 'Something went wrong')
```

Reply:
Yes makes sense. 👍  I just copied it from Microsoft Teams. 😅